### PR TITLE
RDoc-1995 Explained why we see recently deleted files after live-import.

### PR DESCRIPTION
**Pull request by serezhub**
Updated Studio interface images and instructions to include new Tasks tab as of 5.1 in the Cloud>Migration article.

Updated Cloud > New Product article to include Google Cloud Platform option.

### CODE REVIEW

File: Documentation/4.2/Raven.Documentation.Pages/cloud/portal/cloud-portal-products-tab.markdown [hunk 1]
```diff
@@ -31,16 +31,16 @@ The New Product wizard will open, and walk you through four simple stages:
 
 ####1. New Product: Plan
 Use the Plan page to choose your product's **cloud provider**, **region** and **tier**.  
-!["Products Tab Plan"](images\portal-products-tab-plan-001.png "Products Tab Plan")  
+!["Plan Provider Region & Tier"](images\provider-region-tier-selection.png "Plan Provider Region & Tier")
```
[the remark this answers is not in the document]
Comment (reebhub) [reply Documentation/4.2/Raven.Documentation.Pages/cloud/portal/cloud-portal-products-tab.markdown:34]: since the titles are numbered, change the numbers in the drops (and in the related texts) to letters to prevent confusion.

[the remark this answers is not in the document]
Comment (reebhub) [reply Documentation/4.2/Raven.Documentation.Pages/cloud/portal/cloud-portal-products-tab.markdown:34]: match the frames and drops format with the one we use in other places (smaller drops, tighter frame)

File: Documentation/4.2/Raven.Documentation.Pages/cloud/cloud-migration.markdown [hunk 2]
```diff
@@ -9,6 +9,7 @@ cloud servers and between [different RavenDB versions](../migration/client-api/i
 * In this page  
   * [Import From Live RavenDB instance](cloud-migration#import-from-live-ravendb-instance)  
   * [Import From File](cloud-migration#import-from-file)  
+  * [If Files Deleted From Source Database Appear In Destination Database After Live-Import](cloud-migration#if-files-that-were-deleted-from-source-database-appear-in-the-destination-database)
```
[the remark this answers is not in the document]
Comment (serezhub) [reply Documentation/4.2/Raven.Documentation.Pages/cloud/cloud-migration.markdown]: I changed it in my last commit https://github.com/ravendb/docs/pull/1379/commits/c394ab5de198593ef446117c4e95d046af4c5f29

File: Documentation/4.2/Raven.Documentation.Pages/cloud/portal/cloud-portal-products-tab.markdown (hunk 1 above)
[the remark this answers is not in the document]
Comment (serezhub) [reply Documentation/4.2/Raven.Documentation.Pages/cloud/portal/cloud-portal-products-tab.markdown]: Made the changes and re-pushed

File: Documentation/4.2/Raven.Documentation.Pages/cloud/cloud-instances.markdown [hunk 3]
```diff
@@ -40,6 +40,10 @@ to create a **free**-tier instance.
   If your free instance is using more resources than the provided credit, you'll need to pay for these resources or stop your instance. 
   {NOTE/}
 
+    {NOTE: }
+   You can currently locate **Free** Instances only in **Amazon AWS - US East (N. Virginia) - us-east-1** region.  
+  {NOTE/}
+
```
Comment (reebhub) [inline Documentation/4.2/Raven.Documentation.Pages/cloud/cloud-instances.markdown:46]: remove this note. you won't be here tomorrow to change it if other options are available or this one is terminated right?

File: Documentation/4.2/Raven.Documentation.Pages/cloud/portal/cloud-portal-products-tab.markdown [hunk 4]
```diff
@@ -11,56 +11,62 @@ and [Production](../../cloud/cloud-overview#the-production-tier).
 * In this page:  
   * [Provisioning a New Product](../../cloud/portal/cloud-portal-products-tab#provisioning-a-new-product)  
   * [Managing an Existing Product](../../cloud/portal/cloud-portal-products-tab#managing-an-existing-product)  
-     - [The General tab: Change Instance Type and Storage](../../cloud/portal/cloud-portal-products-tab#manage-product-the-general-tab)  
-     - [The Security tab: Your Certificate and Allowed IPs](../../cloud/portal/cloud-portal-products-tab#manage-product-the-security-tab)  
-     - [The Danger Zone tab: Terminate your Instance](../../cloud/portal/cloud-portal-products-tab#manage-product--the-danger-zone-tab)  
+     - [The General tab: Change Instance Type and Storage](../../cloud/portal/cloud-portal-products-tab#e.-manage-product-the-general-tab)
```
Comment (reebhub) [inline Documentation/4.2/Raven.Documentation.Pages/cloud/portal/cloud-portal-products-tab.markdown:14]: references in the contents should have the same name as the title you're linking to.

File: Documentation/4.2/Raven.Documentation.Pages/cloud/portal/cloud-portal-products-tab.markdown [hunk 5]
```diff
@@ -11,56 +11,62 @@ and [Production](../../cloud/cloud-overview#the-production-tier).
...
+The New Product wizard will open, and walk you through the following simple stages:  
   
-1. **Plan**  
-2. **Billing**  
-3. **Customize**  
-4. **Summary**  
+A. **Plan**  
+B. **Billing**  
+C. **Customize**  
+D. **Summary**
```
Comment (reebhub) [inline Documentation/4.2/Raven.Documentation.Pages/cloud/portal/cloud-portal-products-tab.markdown:30]: your titles below continue with E, F, etc.
either you continue the list (here, or in the relevant panel) or remove the letters and references to them
it may also be helpful if the references were links, if they already exist.

File: Documentation/4.2/Raven.Documentation.Pages/cloud/portal/cloud-portal-products-tab.markdown [hunk 6]
```diff
@@ -109,38 +120,45 @@ You can set it using its three tabs:
...
 
-#### 1. Manage Product: The General tab  
+#### E. Manage Product: The General tab  
 You can view your configuration and change your product's instance type and storage size here.  
-!["Manage Product tab"](images\portal-products-005-manage-general.png "Manage Product tab")  
+
+!["Figure 9 - Manage Product tab"](images\portal-products-005-manage-general.png "Figure 9 - Manage Product tab")
 
 * **Change Instance Type**  
   Click this button to scale your product up or down by choosing yourself another configuration.
```
Comment (reebhub) [inline Documentation/4.2/Raven.Documentation.Pages/cloud/portal/cloud-portal-products-tab.markdown:129]: what button?
a reader needs to search the image and find what you mean.
please add frames wherever is needed, numbered if there are multiple options (as we always do), so no one will have to search an image for what you mean.

File: Documentation/4.2/Raven.Documentation.Pages/cloud/portal/cloud-portal-products-tab.markdown [hunk 7]
```diff
@@ -109,38 +120,45 @@ You can set it using its three tabs:
 
 ---
 
-#### 1. Manage Product: The General tab  
+#### E. Manage Product: The General tab  
 You can view your configuration and change your product's instance type and storage size here.  
-!["Manage Product tab"](images\portal-products-005-manage-general.png "Manage Product tab")  
+
```
Comment (reebhub) [inline Documentation/4.2/Raven.Documentation.Pages/cloud/portal/cloud-portal-products-tab.markdown:125]: please separate the images from each other with "---". 
it's hard to follow what comment refers to what image.

File: Documentation/5.1/Raven.Documentation.Pages/cloud/cloud-migration.markdown [hunk 8]
```diff
@@ -0,0 +1,138 @@
...
+Each server's Studio is available at it [Portal](../cloud/portal/cloud-portal#cloud-portal)'s 
+[Product tab](../cloud/portal/cloud-portal-products-tab#cloud-account-portal-products).  
+
+!["Server URLs"](images\migration-001-urls.png "Server URLs")
+
+---
+
+In each studio, click **Manage certificates**.  
+
+!["Manage Certificates"](images\migration-002-manage-certificates.png "Manage Certificates")
```
Comment (reebhub) [inline Documentation/5.1/Raven.Documentation.Pages/cloud/cloud-migration.markdown:33]: fix the frames style and precisions wherever is needed.

File: Documentation/4.2/Raven.Documentation.Pages/cloud/cloud-instances.markdown (hunk 3 above)
Comment (serezhub) [reply Documentation/4.2/Raven.Documentation.Pages/cloud/cloud-instances.markdown:46]: Good point.  Done

File: Documentation/4.2/Raven.Documentation.Pages/cloud/portal/cloud-portal-products-tab.markdown (hunk 4 above)
Comment (serezhub) [reply Documentation/4.2/Raven.Documentation.Pages/cloud/portal/cloud-portal-products-tab.markdown:14]: done

File: Documentation/4.2/Raven.Documentation.Pages/cloud/portal/cloud-portal-products-tab.markdown (hunk 5 above)
Comment (serezhub) [reply Documentation/4.2/Raven.Documentation.Pages/cloud/portal/cloud-portal-products-tab.markdown:30]: Indeed.  Done

File: Documentation/4.2/Raven.Documentation.Pages/cloud/portal/cloud-portal-products-tab.markdown (hunk 6 above)
Comment (serezhub) [reply Documentation/4.2/Raven.Documentation.Pages/cloud/portal/cloud-portal-products-tab.markdown:129]: Good catch. That isn't my writing.  I changed it to 
"  Use sliders to set your desired configuration.  This can be changed later as you need to scale to changing data processing needs.  "

File: Documentation/4.2/Raven.Documentation.Pages/cloud/portal/cloud-portal-products-tab.markdown (hunk 7 above)
Comment (serezhub) [reply Documentation/4.2/Raven.Documentation.Pages/cloud/portal/cloud-portal-products-tab.markdown:125]: Done

File: Documentation/4.2/Raven.Documentation.Pages/cloud/cloud-instances.markdown (hunk 3 above)
Comment (serezhub) [reply Documentation/4.2/Raven.Documentation.Pages/cloud/cloud-instances.markdown:46]: True.  Removed.

File: Documentation/4.2/Raven.Documentation.Pages/cloud/portal/cloud-portal-products-tab.markdown (hunk 4 above)
Comment (serezhub) [reply Documentation/4.2/Raven.Documentation.Pages/cloud/portal/cloud-portal-products-tab.markdown:14]: ok

File: Documentation/4.2/Raven.Documentation.Pages/cloud/portal/cloud-portal-products-tab.markdown (hunk 5 above)
Comment (serezhub) [reply Documentation/4.2/Raven.Documentation.Pages/cloud/portal/cloud-portal-products-tab.markdown:30]: Removed E.F..
Make them all into links

File: Documentation/4.2/Raven.Documentation.Pages/cloud/portal/cloud-portal-products-tab.markdown (hunk 6 above)
Comment (serezhub) [reply Documentation/4.2/Raven.Documentation.Pages/cloud/portal/cloud-portal-products-tab.markdown:129]: Changed wording.  They weren't my words, but you're right...it needed to be written better.

File: Documentation/4.2/Raven.Documentation.Pages/cloud/portal/cloud-portal-products-tab.markdown (hunk 7 above)
Comment (serezhub) [reply Documentation/4.2/Raven.Documentation.Pages/cloud/portal/cloud-portal-products-tab.markdown:125]: You're right.  I put lines.  I'm considering putting each pic into a note with its texts to make it easier.

File: Documentation/5.1/Raven.Documentation.Pages/cloud/cloud-migration.markdown (hunk 8 above)
Comment (serezhub) [reply Documentation/5.1/Raven.Documentation.Pages/cloud/cloud-migration.markdown:33]: No snag file here to edit.  Going to try to get into cloud studio to get new image.

**Review by reebhub — APPROVED** [review]
